src/components/Card/Card.css

```diff
@@ -1,31 +1,69 @@
-a {
-  text-decoration: none;
+.card-container {
+  position: relative;
   display: flex;
-  flex-direction: column;
-  align-items: center;
+  flex-direction: row;
+  justify-content: center;
 }
 
-a:hover {
-  text-decoration: underline;
+.article-image-container {
+  position: relative;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
 }
 
-a > .article-image {
-  height: 30vh;
-  width: auto;
-  border: 1px solid white;
-  box-shadow: 3px 3px 3px black;
-  /* margin-top: 5%; */
+.article-image {
+  width: 80%;
+  max-height: 30rem;
+  padding: .3rem;
+  border: 1px solid black;
+  margin-top: 1rem;
+  background-color: white;
+}
+
+.article-title {
+  margin-bottom: 1rem;
+  margin-top: 1.7rem;
+  font-size: 1.5rem;
+  width: 50%;
+  color: black;
+  word-wrap: normal;
+}
+
+.news-card {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin-bottom: 1rem;
+  align-items: center;
+  width: 90%;
+  object-fit: contain;
+  background-color: #f4f7fc;
+}
+
+.topic-text {
+  position: absolute;
+  color: #132741;
+  font-style: italic;
+  left: 12%;
+  margin-top: .5rem;
+  margin-bottom: .5rem;
 }
 
 .article-sentiment {
   position: absolute;
+  height: 1.6rem;
+  width: 1.5rem;
+  font-size: 1.5rem;
   background-color: #0db6e5;
+  color: white;
   text-shadow: 1px 1px 1px black;
   padding: .5rem .5rem .3rem .5rem;
-  border: 1px solid white;
-  border-radius: .5rem;
-  top: -3%;
-  left: 0%;
+  border: 2px solid white;
+  border-radius: 50%;
+  top: 3%;
+  left: 12%;
 }
 
 .green {
```

Comment:
I think the article sentiment colors are still causing contrast errors, would having the font of the sentiment score be black fix the contrast error?

Reply:
I believe it would, but I just darkened the shades of the color indicators and it removed the contrast errors in WAVE. Would that work instead?